charts/databend-query/values.yaml

```diff
@@ -93,14 +98,6 @@ config:
 
   # [meta]
   meta:
-    # If databend-meta is hosted in the same cluster as databend-data, you can enable this to generate endpoints in a K8s native way.
```

Comment:
Why deleting these?

Reply:
oops... didn't notice the update patch, reverted those parts.